library/src/main/res/values/strings.xml

```diff
@@ -38,6 +38,13 @@
     <string name="chucker_share">Share</string>
     <string name="chucker_share_as_text">Share as text</string>
     <string name="chucker_share_as_curl">Share as curl command</string>
+    <string name="chucker_save">Save body to file</string>
+    <string name="chucker_empty_request_body">The request\'s body is empty</string>
+    <string name="chucker_empty_response_body">The response\'s body is empty</string>
+    <string name="chucker_save_is_only_supported_on_19">This feature is currently available only for Android 4.4+</string>
```

Comment:
I guess, this is redundant now, because users won't see an option on devices with older versions of Android.